Java/other/ArrayProductExceptSelf.java

```diff
@@ -0,0 +1,96 @@
+/*
+Given an array arr[] of n integers, construct a Product Array prod[] (of same size) such that prod[i] is equal to the product of all the elements of arr[] except arr[i]
+*/
```

Comment:
- Don't write in such a way it doesn't look good 
- Add a beginner-friendly description at the top